src/approval/simple_multisig.rs

```diff
@@ -0,0 +1,364 @@
+//! Simple multi-signature wallet component. Generic over approvable actions.
+//! Use with NativeTransactionAction for multisig over native transactions.
+
+use std::marker::PhantomData;
+
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    env, AccountId,
+};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use super::{ActionRequest, ApprovalConfiguration};
+
+/// An AccountApprover gatekeeps which accounts are eligible to submit approvals
```

Comment:
Should this say _Account**Authorizer**_ instead of _Account**Approver**_?

Reply:
Yes, yes it should.